docker-compose-distributed.yml

```diff
@@ -0,0 +1,90 @@
+version: '3'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:5.4.0
+    networks:
+      kafka_distributed_network:
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      zk_id: '1'
+    # ports:
+    #  - '2181:2181'
+
+  kafka:
+    image: confluentinc/cp-enterprise-kafka:5.4.0
+    networks:
+      kafka_distributed_network:
+        aliases:
+          - kafka
+    ports:
+      - 29092:29092
+    depends_on:
+      - zookeeper
+    environment:
+      KAFKA_ZOOKEEPER_CONNECT: 'zookeeper:2181'
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+      KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR: 1
+      KAFKA_TRANSACTION_STATE_LOG_MIN_ISR: 1
+      KAFKA_LISTENERS: PLAINTEXT://:9092, EXTERNAL_LISTENER://:29092
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:9092, EXTERNAL_LISTENER://localhost:29092
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT, EXTERNAL_LISTENER:PLAINTEXT
+      KAFKA_INTER_BROKER_LISTENER_NAME: PLAINTEXT
+
+  schema-registry:
+    image: confluentinc/cp-schema-registry:5.4.0
+    networks:
+      kafka_distributed_network:
+        aliases:
+          - schema-registry
+    # ports:
+    # - '8081:8081'
```

Comment:
Like above, shouldn't we remove them? :thinking:

Reply:
The same as above